workflows/crisprseq_screening.nf

```diff
@@ -17,7 +17,13 @@ WorkflowCrisprseq.initialise(params, log)
 
 // Set screening parameters and channels
 if (params.library) { ch_library = file(params.library) }
-if (params.crisprcleanr) { ch_crisprcleanr = Channel.value(params.crisprcleanr) }
+if (params.crisprcleanr) {
+    if(params.crisprcleanr.endsWith(".csv")) {
+        ch_crisprcleanr_file = file(params.crisprcleanr)
+    } else {
+        ch_crisprcleanr_value = Channel.value(params.crisprcleanr)
```

Comment:
If I understand from the code below, you can define the same channel name, for example `ch_crisprcleanr`, instead of having two different channels `ch_crisprcleanr_file` and `ch_crisprcleanr_value `